doc/design/diagrams/iceoryx_components_diagram_nextgen_release.puml

```diff
@@ -0,0 +1,119 @@
+@startuml Eclipse iceoryx Component Overview
```

Comment:
Could you please render this to a SVG? You just need to run `tools/generate_plantuml_svg.sh`

Reply:
Done